doc/tutorials/04-lattice_boltzmann/04-lattice_boltzmann_part2.ipynb

```diff
@@ -88,88 +88,97 @@
    "outputs": [],
    "source": [
     "import espressomd\n",
-    "espressomd.assert_features(\"CUDA\")\n",
-    "from espressomd import System, lb\n",
-    "from espressomd.observables import ParticlePositions\n",
-    "from espressomd.accumulators import Correlator\n",
+    "import espressomd.lb\n",
+    "import espressomd.observables\n",
+    "import espressomd.accumulators\n",
     "\n",
-    "import numpy as np\n",
     "\n",
-    "# Constants\n",
-    "loops = 40000\n",
-    "steps = 10\n",
-    "time_step = 0.01\n",
-    "runtime = loops * steps * time_step\n",
-    "box_l = 16\n",
-    "\n",
-    "lb_friction = 1.0  # The friction has been set to be 1.0 as default\n",
-    "                   # Change this value for further simulations\n",
+    "import logging\n",
+    "logging.basicConfig(level=logging.INFO)\n",
```

Comment:
The default settings of the `logging` module were designed with terminals in mind. All logs are streamed to stderr to help filtering logs from print statements with piping and redirects. In Jupyter, stderr is printed on a red background to alert the user about something critical that does not interrupt the flow of the program (flow interruptions like exceptions are formatted differently). In my opinion, user experience would improve if we didn't display the INFO category in red. This can be achieved by:
```python
import sys
import logging
logging.basicConfig(level=logging.INFO, stream=sys.stdout)
```